eng/pipelines/extra-platforms/runtime-extra-platforms-androidemulator.yml

```diff
@@ -118,3 +118,35 @@ jobs:
           parameters:
             creator: dotnet-bot
             testRunNamePrefixSuffix: Mono_$(_BuildConfig)
+
+#
+# Android emulators
+# Build the whole product using CoreCLR and run libraries tests
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    runtimeFlavor: coreclr
+    platforms:
+    - android_x64
+    - android_arm64
```

Comment:
Do we have arm64 emulators setup? I thought that by default these will go to devices, see
https://github.com/dotnet/runtime/blob/a4f9ec2720b744bb66fd71ecad5f6c08e86376eb/eng/pipelines/libraries/helix-queues-setup.yml#L96-L97

Comparing the logs between x64 and am64, I would say that the arm64 are run on devices https://helixr1107v0xdeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-112283-merge-8bd807b9687245838c/Android.Device_Emulator.JIT.Test/1/console.36c43a9c.log?helixlogtype=result.

Reply:
I'm also wondering, if we have the same runs in `runtime.yml` (which runs always), why we need them again in extra-platforms? 

I think we can have just `runtime.yml` for now and when we include full libraries run we should move that to extra-platforms.